app/autodiscovery/providersxml/src/main/java/com/fsck/k9/autodiscovery/providersxml/KoinModule.kt

```diff
@@ -0,0 +1,8 @@
+package com.fsck.k9.autodiscovery.providersxml
+
+import org.koin.dsl.module
+
+val autodiscoveryProvidersXmlModule = module {
+    factory { ProvidersXmlProvider(get()) }
+    factory { ProvidersXmlDiscovery(get(), get()) }
```

Comment:
When touching Koin module definitions please use named arguments for new or modified entries.